src/main/java/io/swagger/swaggerhub/plugin/requests/SaveSCMPluginConfigRequest.java

```diff
@@ -35,7 +36,7 @@
     private String host;
 
     private SaveSCMPluginConfigRequest(Builder builder) {
-        super(builder.api, builder.apiOwner, builder.version, builder.oas);
+        super(builder.definitionType, builder.api, builder.apiOwner, builder.version, builder.oas);
```

Comment:
Currently SwaggerHub doesn't support plugin integration's for domains, therefore does it make sense to pass a definition type of API as part of this constructor?

Reply:
Makes sense! I'll move down from AbstractSwaggerHubRequest to SwaggerHubRequest instead.